src/lib/components/Newsletter/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @fileoverview Element that renders copyable code.
+ */
+
+import {BaseElement} from '../BaseElement';
+import {debounce} from '../../utils/debounce';
+
+/**
+ * Renders code block that can easily be copied.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class Newsletter extends BaseElement {
+  constructor() {
+    super();
+    this.adjustIFrame = debounce(this.adjustIFrame.bind(this), 100);
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.iframe = this.querySelector('iframe');
+    this.adjustIFrame();
+    window.addEventListener('resize', this.adjustIFrame);
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    window.removeEventListener('resize', this.adjustIFrame);
+  }
+
+  adjustIFrame() {
+    if (this.iframe.scrolling !== 'no') {
```

Comment:
since we control adding the iframe could we add it with scrolling already set to no?

Reply:
I thought it would be good to default scrolling to yes and then with js set it to no. The resizeing of the height wont work if JS is disabled, that way users can still scroll through the content with JS off.